### PR TITLE
Init file

### PR DESCRIPTION
**Pull request by JessicaLim8**
### New init command
Allows users to create a new note using `note init -f <filename>` to solve issue #2 

#### Capabilities

- Will check to see if the file already exists
- Will create a file named via the filename tag if it exists
- If no flag exists, a brand new file will be made

### CODE REVIEW

File: util/jsonCheck.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 module.exports = (filename) => {
+  if (!filename) {
+    return filename;
```
Comment (JonathanGin52) [inline util/jsonCheck.js]: This will always return undefined

File: util/jsonCheck.js [hunk 2]
```diff
@@ -1,4 +1,9 @@
 module.exports = (filename) => {
+  // Returns if filename is invalid
+  if (!filename) {
+    return;
+  }
+  // Adds a .json if the file deosnt have a .json
   if (!filename.includes('.json')) {
```
Comment (JonathanGin52) [inline util/jsonCheck.js]: you can remove the first if statement entirely by changing the second if statement to be:
```suggestion
   if (filename && !filename.includes('.json')) {
```

**Review by JonathanGin52 — COMMENTED** [review]
Suggestion

**Comment by JessicaLim8** [comment]
Followed your suggestion

File: util/jsonCheck.js [hunk 3]
```diff
@@ -1,10 +1,6 @@
...
-  // Returns if filename is invalid
-  if (!filename) {
-    return;
-  }
   // Adds a .json if the file deosnt have a .json
-  if (!filename.includes('.json')) {
+  if (filename && !filename.includes('.json')) {
     return filename + '.json';
   }
   return filename;
```
Comment (JonathanGin52) [inline util/jsonCheck.js:6]: useless return

Comment (JessicaLim8) [reply util/jsonCheck.js:6]: whats a useless return

Comment (JonathanGin52) [reply util/jsonCheck.js:6]: the final one

Comment (JonathanGin52) [reply util/jsonCheck.js:6]: wait, nvm, I forgot it could already include '.json'

**Review by JonathanGin52 — APPROVED** [review]

**Pull request merged**